### DISCUSSION TITLE
Feedback from my tests with 2.7 (dev branch) on Prusa XL - it works!

### DISCUSSION THREAD

**Discussion by DoubleStrike** [supermerill/SuperSlicer#4481] [General]
A couple basic points first: **No, I will not be posting a link to my binary build of SuperSlicer 2.7** as this is not ready for non-dev use yet.  Second, if you want to build it yourself, the instructions are complete and accurate [here](https://github.com/supermerill/SuperSlicer/discussions/4090), but please don't bother supermerill with "i'm not a dev but how can I compile this" - that doesn't help anyone. **If you don't know how to setup and configure a build environment, this is likely not ready/stable enough for your use yet - be patient please.**

1. I have gotten the `dev` branch up and running and have been doing tests with my Prusa XL.  One lesson is that since you have to setup a profile from scratch, it is REALLY important to go thru and check each and every setting and config.  There are lots of little landmines that will lead to a program crash or a slicing failure (or potentially a print problem).  I'd be happy to post a profile for other folks looking to do their own private pre-Alpha testing, **but I don't trust my own profile yet.** 
2. I was able to do a few multi-tool prints, but with all tools using PLA for now.  They work well (remember to turn on the wipe tower, as it defaults to off unlike PrusaSlicer).
    - I did notice when it is on, SuperSlicer does a weird thing - it will do small purge/wipe at the front left corner of the print bed, then print the wipe tower.  (This is in addition to the normal "purge a line into thin air" thing the XL normally does at the very start of the print.)  Maybe this is normal for other printers, but the XL gcode normally doesn't do this. Not a huge deal, as it's a tiny amount of filament used, but I would prefer it to not contaminate my work surface in areas I don't need to print or risk damaging/lifting the screen printing at the corner of the sheet.
    - Also, at the current time, the "Tool" view is broken in the slicing view, but luckily the "Filament" view basically is the same thing.

4. My next series of tests will be multi-material using PETG supports with zero-gap under PLA parts.  Traditionally this has worked beautifully, so I will update this post with my results on that front.
5. After that, I'll start doing some other functional tests to try to see how the gcode emitted by SS varies from PS for the same parts.

### Again, this is more to keep people apprised of how it feels to use a 2.7.x build of SS on a recent model printer.  I'm not going to start submitting bugs, etc., as that will be more annoying for supermerill than anything, and I do not want to cause him extra/redundant work.

**Comment by darco415** [comment]
Thanks for the update since we hear nothing about progress here

**Comment by Super-Matter** [comment]
i am currently in the process of building ss 2.7, will fork dev_nightly branch and post binaries for windows as releases

**Reply by DoubleStrike** [reply]
I'd just recommend putting some sort of warning telling people to not post bugs/issues/discussion threads yet, as this is still not fully merged/functional code - it is HIGHLY unstable in its current form and will crash in unexpected ways, and often.  That was the main reason I didn't post binaries - anyone who can't build it themselves yet is not going to be happy with the results.  See the comments [here ](https://github.com/supermerill/SuperSlicer/discussions/4090#discussioncomment-8777153)for more context of why I'm saying the same thing.

**Comment by DoubleStrike** [comment]
Just a follow-up to say I have stopped my testing since I was running into a lot of unexplained crashes while tweaking slicing settings. My last testing was as of mid-December 2024. For now, I have removed my local clone of the repo and will restart from scratch when I see some formal update from Merrill about his timeline and progress - doesn't make sense to just test randomly at this point.
